_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -34,7 +35,7 @@ This rule applies to each [link history state][] of any [semantic link][], for w
 For each test target, there exists at least one [adequate inspection state][] and at least one [visible][] [inclusive descendant][] or [ancestor][] element that is not an [ancestor][] of the **non-link line text**, such that at least one of the following is true:
 
 - **distinguishing style**: the [visible][] [inclusive descendant][] or the [ancestor][] element has a [distinguishable style][] from each element containing **non-link line text**; or
-- **distinguishing content**: the [visible][] [inclusive descendant][] has adjacent content (such as an image or text) that indicates the test target is a link.
+- **distinguishing content**: the [text nodes][text node] of the [visible][] [inclusive descendant][] or the [text nodes][text node] [visible][] of the [inclusive sibling][] include the word "link" (or a translation of "link" to the [default page language][]).
```

Comment:
Major point: I'm really not sure about this change…

I'm not sure that the new Failed Example 3 (`<p>click here: <a …>foo</a></p>`) is failing the SC since "click here" can be understood as "this is a link". And `<p>to read about W3C <a href="…">click here</a></p>` would also fail the rule but I'd say that most people would understand "click here" as "this very text is a link".

Similarly, the old Passed Example 11 (`<p>Read about <img src="link-icon" /><a href="…">W3C</a></p>`) is now failing the rule, but again I'm not sure it is failing the SC.
(actually, since it was a Passed Example in the previous version of the rule, we were all convinced it was passing the SC because of the [Consistency With Accessibility Requirements](https://github.com/act-rules/act-rules.github.io/blob/develop/pages/design/test-cases.md#consistency-with-accessibility-requirements) of our test cases).


Also, I'm not sure this change really lessen the "encourage bad link names" effects that @mitchellevan was referring in https://github.com/act-rules/act-rules.github.io/pull/1010#discussion_r517267821 because this sounds like each link should have the word "link" next to it.

I think we can definitely add a Background note in the line of "yeah, `click this link` passes the rule but fails 2.4.9 and maybe 2.4.4."

Reply:
Looking back at the change, I don't think I'm too comfortable with it either. Requiring the text to use the work "link" is too restraining. 

Given the difficulties in arriving at a consensual definition for "distinguishing content" (both for what is text or icons that distinguish something is a link and where can they be placed), how would everyone feel with removing that from the expectation and creating an assumption that there is no text or icons or other means that identify the link as such?

Reply:
That would definitely work.
* I don't know how broad of an assumption this would end up being, that is what proportion of web pages or targets would violate it. If the assumption is too broad, that might weaken the rule quite a bit. OTOH, we don't really know that now and having implementations of the rule might be the best way to gather data…
* At the same time, I'd say that **distinguishing style** is a better practice than **distinguishing content** in any case… So only keeping the former might just end up having a stricter rule (with an assumption to keep the mapping), which is rather good.

So, yeah, I think this is worth trying and maybe we'll need to re-evaluate once we have data from tool vendors…

Reply:
Removed it